portal/src/App.vue

```diff
@@ -24,6 +25,7 @@ useHead({
 		class: "theme-dark",
 	},
 });
+
```

Comment:
Remove extra line. Were these added by the formatter?

Reply:
These were added by me, I removed them. Not sure why I did it.